resources/newssite/news-nuxt/dist/index.html

```diff
@@ -3,6 +3,6 @@
 <head><meta charset="utf-8">
 <title>The Daily Broadcast</title>
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<meta name="description" content="A news site developed with Nuxt."><link rel="modulepreload" as="script" crossorigin href="./_nuxt/entry.eed2f10a.js"><link rel="preload" as="style" href="./_nuxt/entry.762f5ea3.css"><link rel="prefetch" as="script" crossorigin href="./_nuxt/error-component.a9992003.js"><link rel="stylesheet" href="./_nuxt/entry.762f5ea3.css"></head>
-<body ><div id="__nuxt"></div><script type="application/json" id="__NUXT_DATA__" data-ssr="false">[{"_errors":1,"serverRendered":2,"data":3,"state":4},{},false,{},{}]</script><script>window.__NUXT__={};window.__NUXT__.config={public:{},app:{baseURL:"./",buildAssetsDir:"/_nuxt/",cdnURL:"./"}}</script><script type="module" src="./_nuxt/entry.eed2f10a.js" crossorigin></script></body>
+<meta name="description" content="A news site developed with Nuxt."><link rel="modulepreload" as="script" crossorigin href="./_nuxt/entry.34ceb9be.js"><link rel="preload" as="style" href="./_nuxt/entry.762f5ea3.css"><link rel="prefetch" as="script" crossorigin href="./_nuxt/error-component.004af810.js"><link rel="stylesheet" href="./_nuxt/entry.762f5ea3.css"></head>
```

Comment:
Unrelated to this change, but I seem to remember some issues with modulepreload causing network traffic during measured time in the editors workload https://github.com/WebKit/Speedometer/pull/81#issuecomment-1462599849. Would have to take a closer look at profiles to see if this is happening here as well.

Reply:
Oh, interesting. We should look into that.